.appveyor.yml

```diff
@@ -3,16 +3,16 @@ clone_depth: 10
 version: "{build}"
 
 environment:
+  fast_finish: true
   matrix:
-    - nodejs_version: "6.9.1"
-      platform: x86
-    - nodejs_version: "7"
+    - nodejs_version: "6"
```

Comment:
ideally i do, but i'm not sure the delay is worth it.

appveyor is significantly slower of a VM than our travis bots, and they don't have a reasonable timeout on inactivity. which means they're very regularly our last CI to finish.

so this evens the playing field a bit, and i don't think missing out on node7/windows is all that bad.